docs/workflow/testing/mono/testing.md

```diff
@@ -1,54 +1,83 @@
-
+    * Build and run all runtime tests. From the root directory run the following command:
 ```
-dotnet build /t:RunCoreClrTest /p:CoreClrTest="<TestName>" $(REPO_ROOT)/src/mono/mono.proj
+cd src/mono/netcore
+make run-tests-coreclr-all
 ```
 
-## Running Library Tests
-Running library tests against Mono is straightforward regardless of configuration.  Simply run the following commands:
+* Build and run runtime tests for Webassembly. From the root directory run the following command:
+```
+src/tests/build.sh -skipstressdependencies -excludemonofailures os Browser wasm <Release/Debug>
+```
+    * From the last few lines of the build log, you will see something like this
+```
+--------------------------------------------------
+ Example run.sh command
 
-1. Build and set the RuntimeFlavor
+ src/tests/run.sh --coreOverlayDir=<repo_root>artifacts/tests/coreclr/Browser.wasm.Release/Tests/Core_Root --testNativeBinDir=<repo_root>/artifacts/obj/coreclr/Browser.wasm.Release/tests --testRootDir=<repo_root>/artifacts/tests/coreclr/Browser.wasm.Release --copyNativeTestBin Release
+--------------------------------------------------
+```
+    * To run the tests, copy that command and add `wasm` at the end.
 
-```bash
-./build.sh /p:RuntimeFlavor=mono
+* Build and run Runtime Tests for Android x64. From the root directory run the following command:
+```
+src/tests/build.sh -skipstressdependencies -excludemonofailures os Android x64 <Release/Debug>
 ```
-or on Windows
-```bat
-build.cmd /p:RuntimeFlavor=mono
+    * From the last few lines of the build log, you will see something like this
 ```
+--------------------------------------------------
+ Example run.sh command
 
-2. cd into the test library of your choice (`cd src/libraries/<library>/tests`)
+ src/tests/run.sh --coreOverlayDir=<repo_root>artifacts/tests/coreclr/Browser.wasm.Release/Tests/Core_Root --testNativeBinDir=<repo_root>/artifacts/obj/coreclr/Browser.wasm.Release/tests --testRootDir=<repo_root>/artifacts/tests/coreclr/Browser.wasm.Release --copyNativeTestBin Release
+--------------------------------------------------
+```
+    * To run the tests, copy that command and add `Android` at the end.
+
+* For more details about the nature of the runtime tests, please refer to [Coreclr testing documents](../coreclr)
+
+## Running Library Tests
+* Build and run library tests against Mono JIT:
 
-3. Run the tests
+1. cd into the test library of your choice (`cd src/libraries/<library>/tests`)
+
+2. Run the tests
 
 ```
-dotnet build /t:Test /p:RuntimeFlavor=mono
+dotnet build /t:Test /p:RuntimeFlavor=mono /p:Configuration=<Release/Debug>
 ```
 
-# Patching Local dotnet (.dotnet-mono)
-Another way to test mono out is by 'patching' a local dotnet with our runtime bits.  This is a good way to write simple
-test programs and get a glimpse of how mono will work with the dotnet tooling.
+* Build and run library tests against Webassembly. See instructions for [Testing Webassembly](../libraries/testing-wasm.md)
```

Comment:
I don't think you have to specify every platform here this way. You can either say "refer to the instructions in [folder]" or just have a line like "Instructions for [wasm](link), [android](link)," etc.

Reply:
Fixed